src/sider/arbeidsuforhet/ForhandsvarseAfterDeadline.tsx

```diff
@@ -14,14 +14,14 @@ import {
 } from "@/data/arbeidsuforhet/arbeidsuforhetTypes";
 
 const texts = {
-  title: "Fristen er utgått!",
+  title: "Fristen er gått ut",
   passertAlert: (sentDate: Date) =>
-    `Forhåndsvarselet som ble sendt ut ${tilLesbarDatoMedArUtenManedNavn(
+    `Fristen for forhåndsvarselet som ble sendt ut ${tilLesbarDatoMedArUtenManedNavn(
       sentDate
-    )} er gått ut! Du kan nå gi avslag på Arbeidsuførhet.`,
+    )} er gått ut. Trykk på Avslag-knappen hvis vilkårene i § 8-4 ikke er oppfylt og rett til videre sykepenger skal avslås.`,
```

Comment:
Skrives det med mellomrom eller paragraf etterfulgt av tall uten mellomrom? 🤔
```suggestion
    )} er gått ut. Trykk på Avslag-knappen hvis vilkårene i §8-4 ikke er oppfylt og rett til videre sykepenger skal avslås.`,
```

Reply:
Basert på [denne](https://lovdata.no/dokument/NL/lov/1997-02-28-19/KAPITTEL_5-4#KAPITTEL_5-4) er det med mellomrom, og tror alle tekstene Ingrid har vært med å utforme har mellomrom 🤔

Reply:
<img width="671" alt="image" src="https://github.com/navikt/syfomodiaperson/assets/37441744/15e115d5-a91a-422f-b637-e2d1b78da85e">

Reply:
Aha! Det visste eg ikkje 😊